### PR TITLE
Change package name case to lowercase, removes glog.

### PR DESCRIPTION
**Pull request by ackleymi**
Brings it in line with go convention.

### CODE REVIEW

**Comment by ackleymi** [comment]
Hmmm looks like I need go 1.11 for even something as simple as this. Will update with a fix.

**Comment by rocketbitz** [comment]
go 1.11 is awesome - you should update 😉

**Comment by ackleymi** [comment]
@rocketbitz all set

File: utils/log/log.go [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
-	"github.com/golang/glog"
+	"go.uber.org/zap"
 )
 
+func init() {
+	logger, _ := zap.NewProduction()
+	zap.ReplaceGlobals(logger)
+}
+
 func Log(level Level, format string, args ...interface{}) {
```
Comment (rocketbitz) [inline utils/log/log.go]: I think it's cleaner if we actually change this to be separate functions with `log.Info`, `log.Warn`, `log.Error` etc. so we don't feel the need for dot imports everywhere.

**Pull request merged**